### PR TITLE
Don't swallow control-C

### LINKED ISSUE #587: promptListInteractive swallows Control-C
If you run the following nim program:

```nim
import nimblepkg/cli
discard promptList(dontForcePrompt, "Choose something?", ["a", "b", "c"])
```

And press Control-C, it will *not* break out of the prompt, but I expect that it *should* stop the program.

I'm on macOS.

```
nimble v0.9.0 compiled at 2018-10-19 22:47:19
git hash: 22aa3c1d1392d7878d039250ecf0ffeadf3ba0cc
```

### PR DESCRIPTION
**Pull request by iffy**
Fixes #587 

But doing `quit(1)` seems like the wrong thing to do.  Is there a better way to throw the error that Control-C normally causes?

### CODE REVIEW

File: src/nimblepkg/cli.nim [hunk 1]
```diff
@@ -218,6 +218,9 @@ proc promptListInteractive(question: string, args: openarray[string]): string =
       of '\r':
         selected = true
         break
+      of '\3':
+        showCursor(stdout)
+        quit(1)
```
Comment (dom96) [inline src/nimblepkg/cli.nim]: You can use ``QuitFailure`` here instead. But what you should really do is raise some exception, ``NimbleError`` should be fine for this.

Comment (iffy) [reply src/nimblepkg/cli.nim]: k, this version throws a NimbleError instead.

File: src/nimblepkg/cli.nim [hunk 2]
```diff
@@ -13,6 +13,7 @@
 #   - Normal for MediumPriority.
 
 import logging, terminal, sets, strutils, os
+import ./version
```
Comment (dom96) [inline src/nimblepkg/cli.nim]: Why is this required? Should it be "common"?

Comment (iffy) [reply src/nimblepkg/cli.nim]: I just grepped for the definition of `NimbleError` and found it in version :)  I'll change it to `common`
